docs/design/vcpu-handling.md

```diff
@@ -157,6 +157,32 @@ docker run --cpus 4 -ti debian bash -c "nproc; cat /sys/fs/cgroup/cpu,cpuacct/cp
 400000  # cfs quota
 ```
 
+## Virtual CPU handling without hotplug
+
+In some cases, the hardware and/or software architecture being utilized does not support
+hotplug. For example, Firecracker VMM does not support CPU or memory hotplug. Similarly,
```

Comment:
You could link to https://github.com/kata-containers/kata-containers/blob/main/docs/hypervisors.md here maybe?

Reply:
Since we don't specifically call out CPU / memory hotplug in the hypervisor document, I'm not sure this would be super helpful here.